scalabel/label/typing.py

```diff
@@ -32,6 +32,34 @@ class Poly2D(BaseModel):
     closed: bool
 
 
+class Node(BaseModel):
+    """Node of a graph."""
+
+    # 2D or 3D coordinates.
+    # in 2D: (x, y), x horizontal, y vertical, (0, 0) top left corner
+    location: Union[Tuple[float, float], Tuple[float, float, float]]
+    category: str
+    visibility: Optional[str] = None
+    type: Optional[str] = None
+    id: Optional[str] = None
```

Comment:
id should probably not be optional, otherwise the Node can't be part of a Graph because Edge cannot connect the node to any other

Reply:
You're right, I made it not optional